packages/js/rollup.config.js

```diff
@@ -42,7 +44,17 @@ export default [
   // Server build
   {
     input: 'build/src/server.js',
-    external: ['http', 'https', 'url', 'os', 'fs', 'util', 'domain', 'async_hooks'],
+    external: [
+      'http',
+      'https',
+      'url',
+      'os',
+      'fs',
+      'util',
+      'domain',
+      'async_hooks',
+      'cosmiconfig',
```

Comment:
I'm not entirely sure about this.
My reasoning is:
- this is the server bundle, we want to transform to multiple targets (formats),
- but it can require external modules (i.e. from node_modules)

However, we want the browser bundles to be single files so they can be loaded from the CDN.